ci_framework/roles/rhol_crc/tasks/cleanup.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# It actually deletes everything related with the instance:
+# domain and XML definition, network and XML definition, crc instance, ...
+- name: Delete RHOL/CRC instance and associated configuration
+  ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} cleanup"
```

Comment:
the missing newline may create some issues with either pre-commit/yamllint, or ansible-test/sanity. I think some other files are also in the same case.